.azure-pipelines.yml

```diff
@@ -143,6 +147,32 @@ stages:
       iosCertificateFile: $(AppStoreCertificate)
       iosVariableGroup: 'ApplicationTemplate.Distribution.AppStore'
 
+- stage: Build_Canary
```

Comment:
I would add more comments explaining what are the canary steps, imagine someone doesn't know what's a canary, he would need to know this can be removed.

Reply:
Noted I'll comments